application/models/user.go

```diff
@@ -246,6 +246,11 @@ func (u *User) CanEditOrganization(orgId int) bool {
 	return false
 }
 
+// CanEditAllPosts returns true if the given user is allowed to edit any post
```

Comment:
again, this comment is creating more confusion than the code itself (imo)

Reply:
Perhaps. The linter gives a warning for undocumented, exported functions, and I like to clear up those warnings. I think something is better than nothing, in terms of function docs.